src/ber/ber.rs

```diff
@@ -50,13 +50,17 @@ impl debug BerTag {
     NumericString = 0x12,
     PrintableString = 0x13,
     T61String = 0x14,
+    VideotexString = 0x15,
 
     Ia5String = 0x16,
     UtcTime = 0x17,
     GeneralizedTime = 0x18,
 
+    GraphicString = 15, // 0x19
```

Comment:
That looks a typo (should be 25, according to both spec and your comment)
No need to patch/resend, I'll add the fix